tests/test_series.py

```diff
@@ -263,28 +267,42 @@ class MyEnum(Enum):
 
 
 def test_types_fillna() -> None:
-    s = pd.Series([1, np.nan, np.nan, 3])
-    check(assert_type(s.fillna(0), pd.Series), pd.Series)
-    check(assert_type(s.fillna(0, axis="index"), pd.Series), pd.Series)
+    s = pd.Series([1.0, np.nan])
```

Comment:
why not just use `s = pd.Series([1.0, np.nan, np.nan, 3.0])` to make it closer to the original test?

Reply:
I was shortening some of the Series as their length is irrelevant from a typing perspective. Can revert it back.